data/xml/Q17.xml

```diff
@@ -216,102 +232,124 @@
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1064/242</href>
+    <month>December</month>
+    <doi>10.1162/tacl_a_00058</doi>
+    <abstract>Our goal is to construct a domain-targeted, high precision knowledge base (KB), containing general (subject,predicate,object) statements about the world, in support of a downstream question-answering (QA) application. Despite recent advances in information extraction (IE) techniques, no suitable resource for our task already exists; existing resources are either too noisy, too named-entity centric, or too incomplete, and typically have not been constructed with a clear scope or purpose. To address these, we have created a domain-targeted, high precision knowledge extraction pipeline, leveraging Open IE, crowdsourcing, and a novel canonical schema learning algorithm (called CASI), that produces high precision knowledge targeted to a particular domain - in our case, elementary science. To measure the KB’s coverage of the target domain’s knowledge (its “comprehensiveness” with respect to science) we measure recall with respect to an independent corpus of domain text, and show that our pipeline produces output with over 80% precision and 23% recall with respect to that target, a substantially higher coverage of tuple-expressible science knowledge than other comparable resources. We have made the KB publicly available.</abstract>
     <pages>233–246</pages>
-    <url>http://www.aclweb.org/anthology/Q17-1017</url>
+    <url>https://www.aclweb.org/anthology/Q17-1017</url>
   </paper>
 
   <paper id="1018">
     <title>Sparse Coding of Neural Word Embeddings for Multilingual Sequence Labeling</title>
-    <author><first>Gábor</first><last>Berend</last></author>
+    <author><first>Gáabor</first><last>Berend</last></author>
```

Comment:
Is this name in the XML this way? The PDF has just one "a": https://www.aclweb.org/anthology/Q17-1018

Reply:
Sure is.

```xml
<string-name name-style="western"><given-names>Gáabor</given-names><x xml:space="preserve"> </x><surname>Berend</surname></string-name>
```

Reply:
This should be fixed now in the latest release, per the email thread. Or we could just correct this name and others manually here—that's probably easier.